src/seen.coffee

```diff
@@ -39,22 +39,18 @@ class Seen
   constructor: (@robot) ->
     @cache= {}
 
-    @robot.brain.on 'loaded', @load
-    if @robot.brain.data.users.length
```

Comment:
Accio @technicalpickles (not sure if he's back from paternity leave yet? See github/hubot#1210 but also https://github.com/hubot-scripts/hubot-redis-brain/pull/24#issuecomment-268148923)